src/main/java/dan200/computercraft/shared/pocket/apis/PocketAPI.java

```diff
@@ -155,21 +156,21 @@ public void shutdown()
         }
     }
 
-    private static IPocketUpgrade findUpgrade( ItemStack[] inv, int start, IPocketUpgrade previous )
+    private static IPocketUpgrade findUpgrade( NonNullList<ItemStack> inv, int start, IPocketUpgrade previous )
     {
-        for (int i = 0; i < inv.length; i++)
+        for( int i = 0; i < inv.size(); i++ )
```

Comment:
No. If you look further down I'm doing `ItemStack invStack = inv.get( (i + start) % inv.size() );` in order to start from the current slot and loop through the inventory.